routes/middleware/post_place.js

```diff
@@ -0,0 +1,35 @@
+const request = require('request');
+module.exports = (req, res, next) => {
```

Comment:
Add an empty line between lines 1 and 2 for consistency's sake and then it's good to go! :thumb: